src/proto/game_state.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto2";
+
+package TbotsProto;
+
+import "proto/geometry.proto";
+import "proto/vision.proto";
+
+message GameState
+{
+    enum PlayState
+    {
+        PlayState_HALT    = 0;
+        PlayState_STOP    = 1;
+        PlayState_SETUP   = 2;
+        PlayState_READY   = 3;
+        PlayState_PLAYING = 4;
+    }
+    enum RestartReason
+    {
+        NONE           = 0;
+        KICKOFF        = 1;
+        DIRECT         = 2;
+        INDIRECT       = 3;
+        PENALTY        = 4;
+        BALL_PLACEMENT = 5;
+    }
+    enum RefereeCommand
+    {
+        HALT                 = 0;
+        STOP                 = 1;
+        NORMAL_START         = 2;
+        FORCE_START          = 3;
+        PREPARE_KICKOFF_US   = 4;
+        PREPARE_KICKOFF_THEM = 5;
+        PREPARE_PENALTY_US   = 6;
+        PREPARE_PENALTY_THEM = 7;
+        DIRECT_FREE_US       = 8;
+        DIRECT_FREE_THEM     = 9;
+        INDIRECT_FREE_US     = 10;
+        INDIRECT_FREE_THEM   = 11;
+        TIMEOUT_US           = 12;
+        TIMEOUT_THEM         = 13;
+        GOAL_US              = 14;
+        GOAL_THEM            = 15;
+        BALL_PLACEMENT_US    = 16;
+        BALL_PLACEMENT_THEM  = 17;
+    }
+    required PlayState play_state         = 1;
+    required RestartReason restart_reason = 2;
+    required RefereeCommand command       = 3;
+    required BallState ball_state         = 4;
+    required Point ball_placement_point   = 5;
```

Comment:
I think ball_state and ball_placement_point are optional in the GameState class, so they should be optional here too